drivers/sdhc/Kconfig.rcar

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+config MMC_RCAR
+	bool "Renesas Rcar MMC driver"
+	default y
+	depends on DT_HAS_RENESAS_RCAR_MMC_ENABLED
+	select SDHC_SUPPORTS_NATIVE_MODE
+	select REGULATOR
+	select GPIO
+	select SDHC_SUPPORTS_UHS if SDMMC_STACK
+	help
+	  Renesas Rcar MMC driver.
+
+if MMC_RCAR
+
+config RCAR_MMC_DMA_SUPPORT
+	bool "Internal DMA support for Renesas Rcar MMC driver"
+	select CACHE_MANAGEMENT
+	select DCACHE
+	default y
+	help
+	  Internal DMA support for Renesas Rcar MMC driver.
+
+config RCAR_MMC_SCC_SUPPORT
+	bool "Support of SCC"
+	default y
+	help
+	  Enable support of Sampling Clock Controller for Renesas Rcar MMC driver.
+
+if RCAR_MMC_DMA_SUPPORT
+
+config SDHC_BUFFER_ALIGNMENT
+	default 128
```

Comment:
Sounds like devicetree property.

Reply:
`SDHC_BUFFER_ALIGNMENT` is a Kconfig because it is intended to be raised from the default value by the SD host controller driver when needed. The idea for this Kconfig is that it can be used in order to force the SD subsystem to only provide aligned buffers to the SD host controller driver for data transfer. This can be helpful for operations like disk i/o, where we do not guarantee an alignment for buffers.

The reason this setting isn't defined in devicetree is because I view it as a software configuration property, not a hardware one. If we wanted to define SDHC alignment as a host controller property, we could change the SD subsystem to take that approach.